app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable shared among all instances of `Animal`. This means that all animals, regardless of their type, will be added to the same list. If this is intended, it's fine, but if you want separate lists for different types of animals, consider making `alive` an instance variable or using a different structure.